backend/src/main/java/de/webdev/backend/security/config/SecurityConfig.java

```diff
@@ -36,10 +37,20 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
 						authorizeRequests
 								.requestMatchers(HttpMethod.POST, "api/users/register").permitAll()
 								.requestMatchers(HttpMethod.POST, "api/users/login").permitAll()
+								// Members
 								.requestMatchers(HttpMethod.DELETE, apiMembers).hasRole(AppuserRole.ADMIN.name())
 								.requestMatchers(HttpMethod.GET, apiMembers).hasRole(AppuserRole.ADMIN.name())
 								.requestMatchers(HttpMethod.POST, apiMembers).hasRole(AppuserRole.ADMIN.name())
 								.requestMatchers(HttpMethod.PUT, apiMembers).hasRole(AppuserRole.ADMIN.name())
+								.requestMatchers(HttpMethod.DELETE, apiMembers).hasRole(AppuserRole.GROUP1.name())
```

Comment:
if it's the same for all types of requests, you could just not specify it and only write it once? but obv doesn't hurt, and if you do plan to differentiate between different http request methods, then this makes sense
```suggestion
								.requestMatchers(apiMembers).hasRole(AppuserRole.GROUP1.name())
```